generator/src/test/java/com/scottlogic/deg/generator/cucumber/Violation.feature

```diff
@@ -1,22 +1,37 @@
 Feature: The violations mode of the Data Helix app can be run in violations mode to create data
 
-Background:
-     Given the generation strategy is full
-       And there is a field foo
-       And the data requested is violating
-       And the generator can generate at most 5 rows
+  Background:
```

Comment:
Was the white space change in this file intentional?

Reply:
We added the one scenario at the bottom of this feature, and wanted the feature formatted